docs/admin/pool_operations.md

```diff
@@ -286,6 +286,28 @@ The example below shows a rebuild in progress and NVMe space allocated.
     Rebuild busy, 75 objs, 9722 recs
 ```
 
+After experiencing significant failures, the system may retain some suspect
+engines that have been marked as DEAD by the SWIM protocol but were not excluded
+from the system to prevent data loss. An administrator can bring these engines back
```

Comment:
Agreed. Plus, two nit-picking comments below, just for consideration.

"The system may retain" and "from the system" are technically incorrect. Such an engine is not excluded **from the pool**, but may have been excluded from the system. So "the pool" might be more accurate.

"To prevent data loss" could be more accurate as "to prevent potential data inconsistency" or "to preserve data consistency". The foremost purpose of not excluding above RF is to prevent applications from reading potential data that would be impacted if we were to exclude all unavailable engines. The second purpose is to avoid having to reintegrate too many engines to work around certain aspects of the current reintegration protocol.